transcription/claude_sonnet.py

```
@@ -0,0 +1,78 @@
+def encode_image_to_base64(image_path):
+    # Encode an image located at `image_path` into a Base64 string.
+    try:
+        # Open and read the image file in binary mode
+        with open(image_path, "rb") as image_file:
+            # Read the contents of the image file
+            image_data = image_file.read()
+        
+        # Encode the image data into Base64
+        base64_encoded_image = base64.standard_b64encode(image_data).decode('utf-8')
+        
+        return base64_encoded_image
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)